src/plot-data.ts

```diff
@@ -1128,9 +1127,9 @@ export abstract class PlotData {
 
   is_inside_band(real_x, real_y, axis_index): boolean {
     if (this.vertical === true) {
```

Comment:
Could just be `if (this.vertical)`.

On the other hand, consider this syntax (ternary operator) : 

```js
const value = this.vertical ? real_y : real_x
return this.rubber_bands[axis_index].includesValue(value, this.axis_list[axis_index])
```

Reply:
Done and thanks for advice